test/resources/demo-users/user/managed-ns2/rpa.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     release.appstudio.openshift.io/auto-release: 'true'
 spec:
   applications:
-    - test-component
+    - testapp
```

Comment:
Why did you change the application's name?

Reply:
For testing purposes, I made that change. I reverted it.